build.sh

```diff
@@ -208,6 +208,19 @@ isMSBuildOnNETCoreSupported()
         elif [ "$__BuildOS" == "OSX" ]; then
             __isMSBuildOnNETCoreSupported=1
         fi
+    elif [ "$__BuildArch" == "arm" ] || [ "$__BuildArch" == "arm64" ] ; then
+        if [ "$__BuildOS" == "Linux" ]; then
+            if [ "$__DistroName" == "ubuntu" ]; then
+                __isMSBuildOnNETCoreSupported=1
```

Comment:
Assuming all your test devices and those of most consumers' are running Ubuntu on ARM; you can safely remove rest of the changes after this line, until someone actually start using/testing CoreCLR on RHEL, Debian, OS X etc.